paper-typeahead-input.html

```diff
@@ -473,8 +504,8 @@
       }
     },
     _search:function(term){
-      if (term == ""){
-        this._suggestions = [];
+      if (term == "" && !this.isAllowSuggestOnEmptyInput){
```

Comment:
And also note that based on current implementation, the search will go localCandidates, _prefetchedCandidates and remoteCandidates in order. So for you use case, the top 10 will be the first 10 items in localCandidates.